tests/examples/79_delay_warning_threshold.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: 79 delay warning threshold
+  hosts: all
+  delay_warning_threshold: 1 milliseconds
```

Comment:
This is just a test to match 3 events. As we set `delay_warning_threshold: 1 milliseconds`, we will likely observe WARN logs like `r1 is fired with a delay of 3 ms`. Just to manually check the logging.